admin/src/Environment.js

```diff
@@ -49,29 +55,33 @@ class Environment extends Component {
 
     render() {
         const {
-            state: { config, edition, error, loading, newConfig },
+            state: { edition, error, loading, newConfig, filteredConfig },
             effects: { setNewConfig, toggleEdition },
+            setSearch,
         } = this.props;
 
         return (
             <div style={styles.container}>
                 <Card>
-                    <CardText>
-                        {!edition && <RaisedButton label="Edit" primary onClick={toggleEdition} />}
-                        {edition && <RaisedButton label="Save" primary onClick={this.handleSaveClick} />}
-                        {edition && <RaisedButton label="Cancel" onClick={toggleEdition} />}
-                    </CardText>
+                    <CardActions style={styles.actions}>
```

Comment:
The CardActions component has no difference with the CardText component expect for padding